resources/scss/modal.scss

```diff
@@ -39,7 +39,7 @@
     width: 60%;
```

Comment:
if this file will use in all modal, you can file name this modal.scss but if in specific modal change the file name into the modal_name of functionality.scss

Reply:
This file is used in all modal, so this file name is fine with modal.scss.